src/Admin/UserSimpleAdmin.php

```diff
@@ -54,6 +57,7 @@ protected function configureDefaultSortValues(array &$sortValues): void
     #[\Override]
     protected function configureFormFields(FormMapper $form): void
     {
+        $cgsFeatureDate = $this->featureFlagChecker->isEnabled(TermsOfUseHelper::CGS_FEATURE_FLAG_NAME) ? $this->featureFlagChecker->getEnableDate(TermsOfUseHelper::CGS_FEATURE_FLAG_NAME) : null;
```

Comment:
`$cgsFeatureDate = $this->featureFlagChecker->isEnabled(TermsOfUseHelper::CGS_FEATURE_FLAG_NAME) ?:  null;`